cartridges/int_affirm/cartridge/templates/resources/affirm.properties

```diff
@@ -3,9 +3,12 @@ metadata.platform_affirm=22.2.0
 metadata.merchant.name=Test Merchant
 
 affirm.production.url=https://api.global.affirm.com/api
-affirm.production.js=https://www.affirm.com/js/v2/affirm.js
+affirm.production.js=
```

Comment:
is this supposed to be blank?

Reply:
Yes, we're consciously dropping the URL to the CDN hosted js (and default to the static resource version - as per the Salesforce requirement) but left the field so that we can instruct merchants to manually update the value should they choose to use the remote resource version.

Reply:
got it, that makes sense